test/models/developers/query_path_test.rb

```diff
@@ -2,7 +2,10 @@
 
 class Developers::QueryPathTest < ActiveSupport::TestCase
   test "generates unique URL paths for combinations of role levels, location, and freelance" do
-    paths = Developers::QueryPath.all
+    query_path = Developers::QueryPath.new
+    query_path.instance_variable_set(:@paywalled_search_results, false)
```

Comment:
Let's inject this via a a defaulted initializer instead of reaching in under test.

```ruby
class QueryPath

  private attr_reader :paywalled_search_results

  def initialize(paywalled_search_results: Feature.enabled?(:paywalled_search_results))
    @paywalled_search_results = paywalled_search_results
  end

  # ...
end
```